src/app.js

```diff
@@ -95,15 +97,21 @@ class Application {
           `Run for Postman collection '${collectionName}' started. Using '${reporterType}' reporter.`
         );
 
+        const timeout = req.query.timeout;
+        if (timeout) {
+          logger.info(`Timeout has been explicitly set: ${timeout}ms`);
+        }
```

Comment:
OPT: since we're explicit in the yaml file on the default value (30s), what about setting it explicitely here if it isn't provided (so that e.g. if newmann default changes in the future, our doc would remain valid nevertheless)

Reply:
I'll remove the documentation about the 30s. Postman does not publicly advertise this limit, so I would rather we simply to set the "timeout" if none is provided